### ISSUE TITLE
Add support for offline placeholder images

### ISSUE THREAD

**Issue by tgerulaitis** [meanbee/magento-meanbee-pwa#2]
We should add a placeholder image to return from the service worker when an image request fails (e.g. when offline and the image hasn't been cached). Also add a system configuration option to override the placeholder image with a custom one for the store.

**Comment by thomas-thaulow** [comment]
Did this ever get started on?

**Comment by tgerulaitis** [comment]
@CiaraCode, nope, we're not doing any active development on this extension at the moment. Feel free to pick it up if you wish!

Cheers,
Tomas
